src/riak_cs_gc_console.erl

```diff
@@ -195,10 +198,40 @@ human_time(Seconds) ->
     Seconds0 = Seconds + ?DAYS_FROM_0_TO_1970*?SECONDS_PER_DAY,
     rts:iso8601(calendar:gregorian_seconds_to_datetime(Seconds0)).
 
-parse_batch_opts([]) ->
-    [];
-parse_batch_opts([Leeway | _]) ->
-    [{leeway, catch list_to_integer(Leeway)}].
+parse_batch_opts([Leeway]) ->
+    try
+        LeewayInt = list_to_integer(Leeway),
+        {ok, [{leeway, LeewayInt}]}
+    catch T:E ->
+            {error, {T, E}}
+    end;
+parse_batch_opts(Args) ->
+    case getopt:parse(batch_options(), Args) of
+        {ok, {Options, _}} -> {ok, convert(Options)};
+        {error, _} = E -> E
+    end.
+
+batch_options() ->
+    [{leeway, $l, "leeway", string, "Leeway Seconds"},
```

Comment:
This was intended for `infinity`, but who runs leeway with `infinity` ? I was blind.